server/db.ts

```diff
@@ -12,7 +13,14 @@ export const connectDB = (Env: Environment) => {
     database: Env.DB_NAME,
     synchronize: Env.DB_SYNCHRONIZE,
     logging: Env.DB_LOGGING,
-    entities: [User, Post, Comment]
+    entities: [User, Post, Comment],
+    migrations: [seed1565945881563]
   };
-  return createConnection(db);
+  return createConnection(db)
+    .then(connection => connection.runMigrations())
+    .then(() => getConnection())
+    .catch(error => {
```

Comment:
You should not need catch here because `start-app` is catching these. Also what about doing this logic like this so you don't have to fetch connection multiple times? 
![image](https://user-images.githubusercontent.com/12229968/63228066-bc7dfe00-c1f6-11e9-9dad-e8ceb2fd93df.png)

Reply:
Oh.. That seems quite clever to use async inside promise! Didn't even cross my mind to do it like that! Very nice and clean solution!